verification/curator-service/ui/cypress/integration/components/AppTest.spec.ts

```diff
@@ -89,8 +89,8 @@ describe('App', function () {
         cy.contains('Create new').should('not.exist');
         cy.contains('Home');
         cy.contains('Linelist');
-        cy.contains('Sources');
         cy.contains('Profile');
+        cy.contains('Sources').should('not.exist');
```

Comment:
Could you add a check for the uploads tab in all these "Homepage with logged in *" tests?

Reply:
sure, done.